### PR TITLE
[FIX] Fix account owner information not showing

### PR DESCRIPTION
**Pull request by Bendakh**
## Description

Add `v2/accounts-information` api call to get the owner information to fix the owner name not showing.

### CODE REVIEW

**Review by qhdinh — APPROVED** [review]

File: .vscode/settings.json
Comment (amarlankri) [inline .vscode/settings.json:1]: I guess you didn't want to commit this file.

**Review by amarlankri — APPROVED** [review]
Thanks @Bendakh!

File: src/aggregator/services/bridge/bridge-v2.utils.ts [hunk 1]
```diff
@@ -119,19 +123,55 @@ const mapUsageType = (isPro: boolean): AccountUsage => (isPro ? AccountUsage.PRO
...
+): number => {
+  const NOT_FOUND: number = -1;
+  let foundIndex = -1;
+
+  for (let i = 0; i < accountsInformation.length; i++) {
+    const accountIndex = accountsInformation[i].accounts?.findIndex((account) => account.id === accountItemId);
+
+    if (accountIndex !== NOT_FOUND) {
+      foundIndex = i;
+      break;
```
Comment (g-ongenae) [inline src/aggregator/services/bridge/bridge-v2.utils.ts:168]: Why not?

```suggestion
      return i;
```

**Review by g-ongenae — APPROVED** [review]

File: .vscode/settings.json
Comment (Bendakh) [reply .vscode/settings.json:1]: Oh yes my bad

**Review by meriamBenSassi — APPROVED** [review]

**Pull request merged**